price-feeder/oracle/client/client.go

```diff
@@ -0,0 +1,248 @@
+package client
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	rpcClient "github.com/cosmos/cosmos-sdk/client/rpc"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
+	tmjsonclient "github.com/tendermint/tendermint/rpc/jsonrpc/client"
+	umeeapp "github.com/umee-network/umee/app"
+	umeeparams "github.com/umee-network/umee/app/params"
+)
+
+// OracleClient defines a structure that interfaces
+// with the umee node
+type (
+	OracleClient struct {
+		ChainID             string
+		KeyringBackend      string
+		KeyringDir          string
+		KeyringPass         string
+		TMRPC               string
+		RPCTimeout          time.Duration
+		OracleAddr          sdk.AccAddress
+		OracleAddrString    string
+		ValidatorAddr       sdk.ValAddress
+		ValidatorAddrString string
+		Encoding            umeeparams.EncodingConfig
+		GasPrices           string
+		GasAdjustment       float64
+		GRPCEndpoint        string
+		KeyringPassphrase   string
+	}
+
+	passReader struct {
+		pass string
+		buf  *bytes.Buffer
+	}
+)
+
+func NewOracleClient(ChainID string,
+	KeyringBackend string,
```

Comment:
```suggestion
func NewOracleClient(
        chainID string,
	keyringBackend string,
```

arguments to functions and methods are not capitalized ;-)